approval_frame/approval_frame_cron

```diff
@@ -1 +1 @@
-0 0 * * * python mailchimp_api.py
+0 0 * * * python manage.py runscript approval_frame.scripts.update_user_subscriptions
```

Comment:
I think you should remove this cron file. As we would need to source the virtual env on the server before running the script, I think we could directly add it in the system/user crontab on the server.